backend/core/src/shared/shared.module.ts

```diff
@@ -12,7 +12,7 @@ import Joi from "joi"
           .default("development"),
         EMAIL_API_KEY: Joi.string().required(),
         EMAIL_FROM_ADDRESS: Joi.string().required(),
-        DATABASE_URL: Joi.string().required(),
+        DATABASE_URL: Joi.string().optional(),
```

Comment:
Was this a temporary change to get things working, or do we want it to be optional?

Reply:
Temporary, I have a local commit to revert that part. I didn't want to keep kicking off builds for every comment I address.